scripts/world_bank/datasets/datasets.py

```diff
@@ -19,21 +19,39 @@
 
 fetch_datasets: Fetches WB dataset lists and resources and writes them to 'output/wb-datasets.csv'
 
-Run: python3 datasets.py mode=fetch_datasets
+Run: python3 datasets.py --mode=fetch_datasets
 
 ============================
 
-download_datasets: Downloads datasets listed in 'output/wb-datasets.csv' to the  'output/downloads' folder.
+download_datasets: Downloads datasets listed in 'output/wb-datasets.csv' to the 'output/downloads' folder.
 
-Run: python3 datasets.py mode=download_datasets
+Run: python3 datasets.py --mode=download_datasets
 
 ============================
 
 write_wb_codes: Extracts World Bank indicator codes (and related information) from files downloaded in the  'output/downloads' folder to 'output/wb-codes.csv'.
 
 It only operates on files that are named '*_CSV.zip'.
 
-Run: python3 datasets.py mode=write_wb_codes
+Run: python3 datasets.py --mode=write_wb_codes
+
+============================
+
+load_stat_vars: Loads stat vars from a mapping file specified via the `stat_vars_file` flag.
+
+Use this for debugging to ensure that the mappings load correctly and fix any errors logged by this operation.
+
+Run: python3 datasets.py --mode=load_stat_vars --stat_vars_file=/path/to/sv_mappings.csv
```

Comment:
we could mention `sample-svs.csv` contains some example statvars

Reply:
Done.